apps/api/src/campaign/campaign.service.ts

```diff
@@ -492,6 +512,11 @@ export class CampaignService {
             billingEmail: paymentData.billingEmail,
           },
         })
+
+        this.webSocketService.sendNotification('successfulDonation', {
+          ...updatedDonation,
+          person: donation.person,
+        })
```

Comment:
here the donation is returned from an Update statement and it will contain all donation fields. Could you please Pick<> only the needed fields for the notification